contrib/sly-package-fu.el

```diff
@@ -23,6 +24,16 @@
 (defvar sly-export-symbol-representation-function
   #'(lambda (n) (format "#:%s" n)))
 
+(defvar sly-import-symbol-package-transform-function
+  'identity
+  "This function is applied to a package name before
+   it is inserted into the defpackage form.
+
+   By default, it is an 'identity, but you may wish
```

Comment:
I don't follow completely here. This is used in the `import-from` form right? If so, then changing the package name from one convention to another would actually create an error in the code, unless the original `package:symbol` designation on which I pressed `C-x i` was already erroneous.

So unless, I'm missing something, I think this functionality is orthogonal to this PR and should be discussed separately.

Reply:
I've used this feature to replace dots with slashes during refactoring of a large system to `package-inferred-system`. It was very convenient to automatically replace dots with slashes when creating `import-from` form.

Here how I've setup this variable in my config:

```lisp
(setq sly-import-symbol-package-transform-function
                  (lambda (package)
                    (replace-regexp-in-string "\\."
                                              "/"
                                              package)))
```